ubuntu-github-actions--20.04/Dockerfile

```diff
@@ -1,14 +1,17 @@
-# Image name: quay.io/opencv-ci/opencv-ubuntu:20.04
+# Version: 4.6.0-1.0.0
+# Image name: quay.io/opencv-ci/opencv-ubuntu-20.04:4.6.0-1.0.0
 
 FROM ubuntu:20.04
 
 RUN \
   apt-get update && \
   DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
     software-properties-common \
-    curl git zip unzip wget rsync mc nano time \
+    curl zip unzip wget rsync mc nano time \
     file \
     && \
+  add-apt-repository ppa:git-core/ppa && \
+  apt-get install -y --no-install-recommends git && \
```

Comment:
Need to update git to the latest version in case of issue when you would need to add all directories by name using `git config --global --add safe.directory <directory>` ([update](https://marc.info/?l=git&m=164989570902912&w=2)).